packages/paste-core/components/combobox/src/Combobox.tsx

```diff
@@ -77,13 +87,61 @@ const Combobox = React.forwardRef<HTMLInputElement, ComboboxProps>(
     } = useComboboxPrimitive({
       initialSelectedItem,
       items,
-      itemToString: itemToString || defaultItemToString,
       onHighlightedIndexChange,
       onInputValueChange,
       onIsOpenChange,
       onSelectedItemChange,
+      ...(itemToString && {itemToString}),
```

Comment:
Can't we just pass it directly? `itemToString,`? Or does it not like the key to be there with a null value? If it doesn't like null values, maybe you can set these to default values when you destructure them like `initialIsOpen = false,` and then it should just work maybe

Reply:
Yeah it doesn't like them to be null, it seems to still ignore the internal implementation that downshift does hence we had to originally supply a `defaultItemToString` so it wouldn't break on us. 

For things like `itemToString`, `inputValue` and `selectedItem` we don't really know what the internal version is 
doing or what it's dependent on. I'm not confident `defaultItemToString` was doing the right thing. If I left `initialSelectedItem`, `selectedItem` and `inputValue` all pass through, and only set `initialSelectedItem` as having a value, it doesn't update the inputValue so it looks broken. If these are only passed when present it works as intended. 

I feel like doing this more closely replicate the default behaviour of downshift